recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     sha256: d2cd8a1de1c498c1832f0765113ad58086d1d3b29bf5d2cc9b1ba60f1919951c
 
 build:
-    number: 0
+    number: 1
```

Comment:
I can revert the build number if you'd prefer. I bumped the number per @jakirkham's suggestion: https://github.com/conda-forge/ncl-feedstock/commit/0d28b20f81f0eb00c2869dbd943aaa89f7babb8f#commitcomment-19875776